app/views/bulk_actions/new.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class='container'>
   <h1>New Bulk Action</h1>
 
-  <%= form_for(@bulk_action) do |f| %>
+  <%= form_for(@bulk_action, html: { 'data-controller' => 'bulk_actions' }) do |f| %>
```

Comment:
Move the StimulusJS controller up in scope so its actions can hide the common fields div.